lib/src/barback/asset_environment.dart

```diff
@@ -135,9 +138,15 @@ class AssetEnvironment {
   /// Constants to passed to the built-in dart2js transformer.
   final Map<String, String> environmentConstants;
 
-  /// The [Transformer]s that should be appended by default to the root
-  /// package's transformer cascade. Will be empty if there are none.
-  final _builtInTransformers = <Transformer>[];
+  /// The [Transformer]s or [AggregateTransformer]s that should be appended by
+  /// default to the root package's transformer cascade. Will be empty if there
+  /// are none.
+  final _builtInTransformers = new Set();
```

Comment:
I removed these entirely actually and just inlined the logic inside of `getBuiltInTransformers`. I was abusing these to have ordering constraints anyways, and they really don't provide value as they are just used in one place.